x-pack/plugins/infra/public/pages/metrics/hosts/components/kpis/hosts_tile.tsx

```diff
@@ -4,22 +4,47 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
+import { i18n } from '@kbn/i18n';
 import React from 'react';
+import { useHostCountContext } from '../../hooks/use_host_count';
+import { useUnifiedSearchContext } from '../../hooks/use_unified_search';
 
-import { useHostsViewContext } from '../../hooks/use_hosts_view';
-import { type ChartBaseProps, MetricChartWrapper } from '../chart/metric_chart_wrapper';
+import { type Props, MetricChartWrapper } from '../chart/metric_chart_wrapper';
 
-export const HostsTile = ({ type, ...props }: ChartBaseProps) => {
-  const { hostNodes, loading } = useHostsViewContext();
+const HOSTS_CHART: Omit<Props, 'loading' | 'value'> = {
+  id: `metric-hostCount`,
+  color: '#6DCCB1',
+  title: i18n.translate('xpack.infra.hostsViewPage.metricTrend.hostCount.title', {
+    defaultMessage: 'Hosts',
+  }),
+  toolTip: i18n.translate('xpack.infra.hostsViewPage.metricTrend.hostCount.tooltip', {
+    defaultMessage: 'The number of hosts returned by your current search criteria.',
+  }),
+  ['data-test-subj']: 'hostsView-metricsTrend-hosts',
+};
+
+export const HostsTile = () => {
+  const { data: hostCountData, isRequestRunning: hostCountLoading } = useHostCountContext();
+  const { searchCriteria } = useUnifiedSearchContext();
+
+  const getSubtitle = () => {
+    const showLimitSubtitle = searchCriteria.limit < (hostCountData?.count.value ?? 0);
```

Comment:
nit: you can maybe directly return here without adding a variable (as it's used only here), the name explains well what it checks tho

Reply:
Yeah. I guess the condition is not so complicated to understand. I'll remove the variable and directly return it.